tvu_net_cron/models/cancel_expired_quotation.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+
+from odoo import models, fields, api 
+from datetime import date
+
+
+class cancelExpiredQuotation(models.Model):
+    _name = "tvu_net_cron.cancel_expired_quotation"
+    _description = ""
+    
+    def _cancel_expired_quotations(self):
```

Comment:
is there a method decorator that should be on this method: https://www.odoo.com/documentation/15.0/developer/reference/backend/orm.html#module-odoo.api